swift/Sources/FlatBuffers/ByteBuffer.swift

```diff
@@ -234,8 +236,10 @@ public struct ByteBuffer {
   mutating func push<T: NativeStruct>(struct value: T, size: Int) {
     ensureSpace(size: size)
     var v = value
-    memcpy(_storage.memory.advanced(by: writerIndex &- size), &v, size)
-    _writerSize = _writerSize &+ size
+    withUnsafeBytes(of: &v) {
+      memcpy(_storage.memory.advanced(by: writerIndex &- size), $0.baseAddress!, size)
+      self._writerSize = self._writerSize &+ size
+    }
```

Comment:
The following works faster than all the new methods that were tested, which simply returns an `UnsafeRawBufferPointer` which then allows us to use the base address to simply copy the data using memcpy. Fast and effective.